tasks/tests.yaml

```diff
@@ -0,0 +1,38 @@
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: 2023-Present The UDS Authors
+
+tasks:
+  - name: test-unit
```

Comment:
with these test tasks... As it is today they wont actually display in `uds run --list`, which i less than ideal from a discoverability standpoint. 

Even if that is acceptable for now (we with maru should be able to facilitate a way to expose certain included tasks in the list command), prefixing each test with `test-` is redundant. The call end up being `uds run test:test-e2e`.

Since its coming form the test task imported as `test:` you can drop the `test-` in all the tests and just then the run commands just become `uds run test:e2e`

Reply:
Understood the `uds run --list` constraint and good point on dropping the `test-` prefix, will update

Reply:
https://github.com/defenseunicorns/maru-runner/issues/75

Reply:
can do a `uds run --list-all` though

Reply:
lol I forget we had added that!

Reply:
It's even in the docs!


![Screenshot 2024-05-10 at 10 37 59 AM](https://github.com/defenseunicorns/uds-cli/assets/42304551/aa5234a0-2df7-4a9d-a2c2-226152d2d032)